build/app/cli.js

```diff
@@ -50,15 +53,73 @@ const aboutJson = () => {
 		.then(about => fs.writeFileSync(path.join(process.cwd(), '/public/__about.json'), JSON.stringify(about, null, 2)));
 };
 
-const buildConfig = require(path.join(process.cwd(), 'n-ui-build.config.js'));
-const cssEntryPoints = Object.keys(buildConfig.entry)
-	.map(target => [target, buildConfig.entry[target]])
-	.filter(([target]) => target.includes('.css'));
+const getCssEntryPoints = () => {
+	const buildConfig = require(path.join(process.cwd(), 'n-ui-build.config.js'));
+	return Object.keys(buildConfig.entry)
+		.map(target => [target, buildConfig.entry[target]])
+		.filter(([target]) => target.includes('.css'));
+};
 
 program.version(nUiVersion);
 
 const webpackConfPath = path.join(__dirname, 'webpack.config.js');
 
+const DEFAULT_REGISTRY_URI = 'https://next-registry.ft.com/v2/services.json';
+
+const triggerMasterBuild = (project) => circleFetch(`/${project}/build`, { method: 'POST', body: JSON.stringify({ branch: 'master' }) });
+
+const lastMasterBuild = (project) => circleFetch(`/${project}/tree/master`);
+
+const getRepoName = ({ repository }) => {
+	if (/https?:\/\/github\.com\/Financial-Times\//.test(repository)) {
+		return repository
+			.replace(/https?:\/\/github\.com\/Financial-Times\//, '')
+			.replace(/\/$/, ''); // trim trailing "/"
+	}
+};
+
+const serves = type => app => type ? app.types && app.types.includes(type) : true;
+
+const getAllAppsToRebuild = async (allApps, registry, servesType) => {
+	let allAppsToRebuild = [];
+
+	const registryData = await fetch(registry).then(fetchres.json);
+	allAppsToRebuild = registryData
+		.filter(serves(servesType))
+		.map(getRepoName)
+		.filter(repo => repo);
+
+	return allAppsToRebuild;
```

Comment:
No need to create and assign the  `allAppsToRebuild` variable, you can just return above =]